class.jetpack-options.php

```diff
@@ -33,6 +33,7 @@ public static function get_option_names( $type = 'compact' ) {
 				'site_icon_id',                // (int)    Attachment id of the site icon file
 				'dismissed_manage_banner',     // (bool) Dismiss Jetpack manage banner allows the user to dismiss the banner permanently
 				'updates',                     // (array) information about available updates to plugins, theme, WordPress core, and if site is under version control
+				'unique_connection',           // (int)  A flag to determine if the site has ever been previously connected to wordpress.com
```

Comment:
I would recommend adding something to this line to explain what `1` means and what `2` means -- there is no clear summation at the moment.